src/FixedDataTable.js

```diff
@@ -535,11 +535,10 @@ var FixedDataTable = createReactClass({
       this._didScrollStart();
     }
 
-    // Cancel any pending debounced scroll handling and handle immediately.
-    this._didScrollStop.reset();
-    this._didScrollStopSync();
-
-    this.setState(this._calculateState(nextProps, this.state));
+    this.setState(this._calculateState(nextProps, this.state), () => {
```

Comment:
so now `didScrollStop` is only called after `setState` actually updates `this.state`
I also removed the reset call here (see #398)

Reply:
nice!  

The comment is out of date now though, right?  We're not cancelling any pending scroll stop, we're just scheduling one.